app/services/customization/customization.ts

```diff
@@ -108,11 +117,16 @@ export class CustomizationService extends PersistentStatefulService<ICustomizati
     const settings = this.getSettings();
 
     return [
-      <IObsInput<boolean>>{
-        value: settings.nightMode,
-        name: 'nightMode',
-        description: $t('Night mode'),
-        type: 'OBS_PROPERTY_BOOL',
+      <IObsListInput<string>>{
+        value: settings.theme,
+        name: 'theme',
+        description: $t('Theme'),
+        type: 'OBS_PROPERTY_LIST',
+        options: [
+          { value: 'night-theme', description: $t('Night (Classic)') },
+          { value: 'day-theme', description: $t('Day (Classic)') },
+          { value: 'spacegray-theme', description: $t('Space Gray') },
```

Comment:
I wish I was joking but this is probably trademarked by Apple lol.

Reply:
whoops meant to take out that option was just used to test

Reply:
Haha was gonna say.  I couldn't see any styles for it anywhere.  Honestly I was looking forward to having a space gray theme in SLOBS.  Sounds amazing 😄

Reply:
it honestly looks really good but i didn't want to include anything without design input